js/adapt-offlineStorage-scorm.js

```diff
@@ -13,6 +13,14 @@ define([
 
   Adapt.offlineStorage.initialize({
 
+    save: function() {
+      // listens to Adapt{tracking:save} event in adapt-stateful-session  
```

Comment:
out of curiosity, is there a specific reason for this to be triggered by the event rather than using the hard API? If so, it would be useful to add a brief explanation here

Reply:
there was some ambiguous and currently out of grasp logic to it, i'll change. :+1: